test/cadet/public/updater_test.exs

```diff
@@ -31,4 +34,71 @@ defmodule Cadet.Public.UpdaterTest do
       assert String.length(token) == 480
     end
   end
+
+  test "Get course id" do
+    use_cassette "updater/get_course_id#1", custom: true do
+      course_id = Updater.get_course_id("T0K3N...")
+      assert String.length(course_id) == 36
+      assert course_id != "00000000-0000-0000-0000-000000000000"
```

Comment:
Should be `refute course_id == "00000000-0000-0000-0000-000000000000"`

Reply:
Done with a2ae3d9.